cartridge-cli.lua

```diff
@@ -2341,6 +2334,52 @@ local function check_if_deprecated_build_flow_is_ised()
     return deprecated_build_flow_is_ised
 end
 
+-- * ------------------- Build dir --------------------
+
+local BUILD_SUBDIRECTORY_NAME = 'cartridge.build'
+
+local function detect_and_check_build_dir()
+    -- By default, application is built in the tmpdir.
+    -- User can specify build directory in TARANTOOL_BUILDDIR env variable.
+    -- There are two cases:
+    -- - specified directory doesn't exists: we just create it and remove after the build
+    -- - directory already exists:
+    --   - ${TARANTOOL_BUILDDIR}/cartridge.build will be the build directory
+    --   - sub-directory cartridge.build is (re)created and used for application build
+    --   - after the build, ${TARANTOOL_BUILDDIR}/cartridge.build  is removed
+
+    local specified_build_dir = os.getenv('TARANTOOL_BUILDDIR')
```

Comment:
Let's validate this path

Reply:
I validate if it's a directory if the path exists.

What kind of validation did you mean?

Reply:
My bad... Yes, that's ok